docs/cookbook.rst

```diff
@@ -166,7 +174,9 @@ your Actors may need one or more of their Abilities
 to do some cleanup.
 You can assign cleanup tasks to your Actor
 using their :meth:`~screenpy.actor.Actor.has_ordered_cleanup_tasks`
-or :meth:`~screenpy.actor.Actor.has_independent_cleanup_tasks` method::
+or :meth:`~screenpy.actor.Actor.has_independent_cleanup_tasks` method:
```

Comment:
I think this one should be a `.`, since the others are, but...

Reply:
fixed